test/test_reads.py

```diff
@@ -1053,14 +1167,34 @@ def testCloseCloseORF(self):
         self.assertFalse(orf.openRight)
         self.assertEqual(orf.id, 'id-[1:7]')
 
+    def testCloseCloseORFWithJunk(self):
+        """
+        An AA read that contains a start and a stop codon should result in
+        just one AAReadORF when its ORFs method is called. The ORF must have
+        the correct start/stop offsets and its left and right sides must be
+        both marked as closed.
+        """
+        OpenORFs = False
+        read = AARead('id', 'AAAMADRADR*')
+        orfs = list(read.ORFs(OpenORFs))
+        self.assertEqual(1, len(orfs))
+        orf = orfs[0]
```

Comment:
This is a good example of something I was saying before. Just use `[orf] = list(read.ORFs(OpenORFs))` so you have one line of code instead of 3.

Reply:
done